libs/Wallet/src/WalletController.cpp

```diff
@@ -0,0 +1,82 @@
+#include "Status/Wallet/WalletController.h"
+#include "NewWalletAccountController.h"
+
+#include <StatusGo/Wallet/WalletApi.h>
+
+#include <StatusGo/Accounts/AccountsAPI.h>
+#include <StatusGo/Accounts/Accounts.h>
+#include <StatusGo/Accounts/accounts_types.h>
+#include <StatusGo/Metadata/api_response.h>
+#include <StatusGo/Utils.h>
+#include <StatusGo/Types.h>
+
+#include <Onboarding/Common/Constants.h>
+
+#include <QQmlEngine>
+#include <QJSEngine>
+
+#include <ranges>
```

Comment:
Oooh :) C++20?

Reply:
Leftover from an experiment that didn't work :(. Not very well supported, unfortunately.